scripts/dts/gen_dts_cmake.py

```diff
@@ -148,6 +148,9 @@ def main():
         for path in compatible2paths[comp]:
             cmake_path = f'{cmake_path}{path};'
 
+        # Remove the last ';'
```

Comment:
I guess you mean the last char, leading would be the first char:
```suggestion
        # Remove the last ';'
```

Reply:
uhm, I did fix that indeed. Do you still see `leading` ?

Reply:
interesting.
It was there when I placed the comment, but looking at my own suggestion it shows that I propose to change the sentence to itself.
consider this solved then.